addon/utils/locale.js

```diff
@@ -12,18 +12,15 @@ export default class Locale {
   //     and use the first value for `rtl` and `pluralForm`
   //  3. Default `rtl` to `false`
   //  4. Ensure `pluralForm` is defined
-  constructor(id, container) {
-    this.id = id;
-    this.container = container;
-    this.rebuild();
-  }
+  this.id = id;
+  this.container = container;
 
-  rebuild() {
```

Comment:
These should be on the prototype so all instances can share one copy of the functions.